src/CalculatorUnitTests/RationalTest.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <CppUnitTest.h>
+#include "Header Files/Rational.h"
+#include "Header Files/RationalMath.h"
+
+using namespace CalcEngine;
+using namespace CalcEngine::RationalMath;
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace CalculatorEngineTests
+{
+    TEST_CLASS(RationalTest)
+    {
+    public:
+        TEST_CLASS_INITIALIZE(CommonSetup)
+        {
+            ChangeConstants(10, 128);
+        }
+        TEST_METHOD(ModuloTest)
+        {
+            Rational rat25(25);
+            Rational ratminus25(-25);
+            Rational rat4(4);
+            Rational ratminus4(-4);
+            Rational res = Mod(rat25, rat4);
+            VERIFY_ARE_EQUAL(res.ToString(10, FMT_FLOAT, 128), L"1");
+            res = Mod(rat25, ratminus4);
+            VERIFY_ARE_EQUAL(res.ToString(10, FMT_FLOAT, 128), L"-3");
```

Comment:
Do we need to be comparing against strings here?  Why not use:
```suggestion
            VERIFY_ARE_EQUAL(res, -3);
```